cmd/rook/version/version.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// Versioncmd is the command for printing version of rook.
```

Comment:
could you make the casing match in the comment? `VersionCmd`

Reply:
Sure, on it.